jobs/pipelines/powervs/odf/zstreams/zstream-odf4.x-powervs-syd05-upgrade/Jenkinsfile

```diff
@@ -0,0 +1,318 @@
+                           scp -i ${WORKSPACE}/deploy/id_rsa -o 'StrictHostKeyChecking=no' root@${BASTION_IP}:/root/ocs-upi-kvm/scripts/tier${TIER_TEST}.log ${WORKSPACE}
+                        '''
+                    }
+                    catch (err) {
+                        echo 'Error ! Tier test script failed!'
+                        env.FAILED_STAGE=env.STAGE_NAME
+                        throw err
+                    }
+                 }
+             }
+        }
+        stage('Capture tier test summary and compress results') {
+            steps {
+                script {
+                    ansiColor('xterm') {
+                        echo ""
+                    }
+                    try {
+                        sh '''
+                              FILE=tier${TIER_TEST}.log
+                              sed -n '/short test summary info/, /Test result:/p' tier${TIER_TEST}.log > tier${TIER_TEST}-summary.txt
+                              echo -n "Upgraded ODF from `echo $OCS_REGISTRY_IMAGE | cut -d ":" -f 2 | tr -dc '[. [:digit:]]'` to ${UPGRADE_OCS_VERSION} && Result:" > slacksummary.txt
+                              sed -n '/short test summary info/, /Test result:/p' tier${TIER_TEST}.log | awk '/passed/||/failed/||/skipped/' | sed 's/=*//g' >> slacksummary.txt
+                              tar -cvzf test-log.tar.gz tier${TIER_TEST}.log
+                        '''
+                    }
+                    catch (err) {
+                        echo 'Error ! Capturing tier test summary failed!'
+                        env.FAILED_STAGE=env.STAGE_NAME
+                        throw err
+                    }
+                }
+            }
+        }
+        stage('Waiting to timeout') {
+            steps {
+                script {
+                    ansiColor('xterm') {
+                        echo ""
+                    }
+                    try {
+                        TIMEOUT_MIN = TIMEOUT.toInteger()
+                        if ( TIMEOUT_MIN != 0 ){
+                            TIMEOUT_SEC=TIMEOUT_MIN*60
+                            echo "HOLDING THE CLUSTER TEAR DOWN FOR ${TIMEOUT_MIN} MINUTES"
+                            sleep TIMEOUT_SEC
+                        }
+                    }
+                    catch (err) {
+                        echo 'Error ! Timeout stage failed!'
+                        env.FAILED_STAGE=env.STAGE_NAME
+                        throw err
+                    }
+                }
+            }
+        } 
+    }
+    post {
+        always {
+            archiveAllArtifacts("test-log.tar.gz", "deploy/results.tar.gz", "tier${TIER_TEST}-summary.txt")
```

Comment:
lets add `deploy/vars.tfvars` also , this will help us to get the OCP variable we used , especially when there is a OCP deployment failure

Reply:
Refer any OCP job

Reply:
Sure, will add